libraries/cyclestreets-view/src/main/res/values-tr/strings.xml

```diff
@@ -63,6 +63,8 @@
   <string name="report_title">Raporunuz</string>
   <string name="report_location_hint">Neredeki konumda raporu taımladın?</string>
 
+  <string name="tap_here">Tap here to show website</string>
```

Comment:
Given that the English text is now `More details...`, I suspect that the other language files should show the same if we haven't translated this string yet.